lastuser_oauth/views/login.py

```diff
@@ -58,7 +59,7 @@ def login():
                 return set_loginmethod_cookie(render_redirect(get_next_url(session=True), code=303),
                     'password')
         except LoginPasswordResetException:
-            return render_redirect(url_for('.reset', expired=1, username=loginform.username.data))
+            return render_redirect(url_for('.reset', nopasswd=1, username=loginform.username.data))
```

Comment:
Let's change this to _not_ pass status in `request.args`, but as a flash message.